examples/real-world/containers/RepoPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { loadRepo, loadStargazers } from '../actions';
+import Repo from '../components/Repo';
+import User from '../components/User';
+import List from '../components/List';
+
+function loadData(props) {
+  const { fullName } = props;
+  props.loadRepo(fullName, ['description']);
+  props.loadStargazers(fullName);
+}
+
+class RepoPage extends Component {
+  constructor(props) {
+    super(props);
+    this.renderUser = this.renderUser.bind(this);
+    this.handleLoadMoreClick = this.handleLoadMoreClick.bind(this);
+  }
+
+  componentWillMount() {
+    loadData(this.props);
```

Comment:
Great job. :+1: 
If you'd like, you can also adopt a convention like https://github.com/acdlite/flux-standard-action (see https://github.com/acdlite/redux-promise as an example middleware using it)